web/js/cloudmap.js

```diff
@@ -37,7 +37,7 @@ $(window).on('load', function(){
     
     $.when(
         $.getJSON("./data.json"),
-        $.getJSON("./style.json")
+        $.getJSON("./../style.json")
```

Comment:
I don't think this is correct as it will break the other visualizations.

Reply:
The problem was that the script errorred that it could not find style.json, therefore i changed the reference.

Reply:
Does it work at your machine without the `...` ? or was there something else i should have done to make it work?